zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py

```diff
@@ -415,7 +422,8 @@ def make_tasks_with_intermediaries(  # pylint: disable=too-many-locals
         op_kwargs: P.kwargs,
     ) -> Tuple[List[mazepa.tasks.Task[R_co]], VolumetricBasedLayerProtocol | None]:
         dst_temp = self._get_temp_dst(dst, idx, self.flow_id)
-        idx_chunks = self.processing_chunker(idx, mode="exact")
+        have_gap = self.gap is not None and self.gap != Vec3D[int](0, 0, 0)
+        idx_chunks = self.processing_chunker(idx, mode="expand" if have_gap else "exact")
```

Comment:
Could you add the comment above this line please:
`# TODO: remove "expand"; see https://github.com/ZettaAI/zetta_utils/issues/648`